tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/trap/TrapOperatorTest.java

```diff
@@ -32,25 +34,42 @@
  */
 public class TrapOperatorTest {
 
+    CompileResult result;
+    CompileResult resultNegative;
+
+    @BeforeClass
+    public void setup() {
+        result = BCompileUtil.compile("test-src/expressions/trap/trap-expr.bal");
+        resultNegative = BCompileUtil.compile("test-src/expressions/trap/trap_negative.bal");
```

Comment:
Since the `trap_negative.bal` file is only used in one test case, shall we move it to the test itself? 
So that we can identify if there is an error occured at compilation separately.